### PR TITLE
fix: invalid pseudo version.

### PR DESCRIPTION
**Pull request by ldez**
The `go.mod` contains some invalid pseudo version:

```
go: github.com/deislabs/smi-sdk-go@v0.0.0-20190621175932-114e91dce170 requires
	k8s.io/apimachinery@v0.0.0-20181127105237-2b1284ed4c93: reading https://proxy.golang.org/k8s.io/apimachinery/@v/v0.0.0-20181127105237-2b1284ed4c93.mod: 410 Gone
	
go: github.com/deislabs/smi-sdk-go@v0.0.0-20190621175932-114e91dce170 requires
	k8s.io/code-generator@v0.0.0-20181116203124-c2090bec4d9b: reading https://proxy.golang.org/k8s.io/code-generator/@v/v0.0.0-20181116203124-c2090bec4d9b.mod: 410 Gone
```

- v0.0.0-20181127105237-2b1284ed4c93 => v0.0.0-20181127025237-2b1284ed4c93
- v0.0.0-20181116203124-c2090bec4d9b => v0.0.0-20181117043124-c2090bec4d9b

For more info see, golang/go#33513

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,17 +20,14 @@ require (
...
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
-	golang.org/x/tools v0.0.0-20190328211700-ab21143f2384 // indirect
-	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 	k8s.io/api v0.0.0-20181213150558-05914d821849
-	k8s.io/apimachinery v0.0.0-20181127105237-2b1284ed4c93
+	k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
 	k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
-	k8s.io/code-generator v0.0.0-20181116203124-c2090bec4d9b // indirect
```
Comment (grampelberg) [inline go.mod]: I'm surprised this dependency is going away.

Comment (ldez) [reply go.mod]: I just did a `go mod tidy`

Comment (ldez) [reply go.mod]: If you want to manage your tools with go modules, take a look to https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md

Comment (ldez) [reply go.mod]: I can create and add a `tools.go` as suggested in the article if your want?

Comment (grampelberg) [reply go.mod]: Ohh, that's how you do it. 👍  `tools.go` sounds like a great idea.

Comment (ldez) [reply go.mod]: Note that the hash c2090bec4d9b doesn't exist:
https://github.com/kubernetes/code-generator/search?q=c2090bec4d9b&type=Commits

Comment (ldez) [reply go.mod]: the date doesn't exist also

**Pull request merged**